### PR TITLE
Purge unused helpers from Helpers.inc

### PR DESCRIPTION
**Pull request by PeXXeR**
Closes: #3529

Signed-off-by: Somesh Sharma <someshsharma0312@gmail.com>

### CODE REVIEW

**Comment by 0xdaryl** [comment]
@genie-omr build all

**Comment by 0xdaryl** [comment]
@genie-omr build xlinux

**Comment by 0xdaryl** [comment]
@genie-omr build xcrlinux

File: compiler/runtime/Helpers.inc [hunk 1]
```diff
@@ -64,113 +61,26 @@ SETVAL(TR_jitProfileParseBuffer,TR_jitProfileBigDecimalValue+1)
...
-SETVAL(TR_DFPToHexFloat,TR_DFPToBinaryLongDouble+1)
-SETVAL(TR_DFPToHexDouble,TR_DFPToHexFloat+1)
-SETVAL(TR_DFPToHexLongDouble,TR_DFPToHexDouble+1)
-SETVAL(TR_jitProfileStringValue,TR_DFPToHexLongDouble+1)
-SETVAL(TR_jitCheckIfFinalizeObject,TR_jitProfileStringValue+1)
-SETVAL(TR_comparePackedDecimal,TR_jitCheckIfFinalizeObject+1)
-SETVAL(TR_COBOLRTMsg, TR_comparePackedDecimal+1)
-SETVAL(TR_packedDecimalPower, TR_COBOLRTMsg+1)
-SETVAL(TR_initialInvokeExactThunk_unwrapper,TR_packedDecimalPower+1)
+SETVAL(TR_emilyCallGlue,TR_referenceArrayCopy+1)
```
Comment (0xdaryl) [inline compiler/runtime/Helpers.inc]: I believe `TR_emilyCallGlue` can go as well.  There is one reference to it in OMR, but that can be deleted.

Comment (PeXXeR) [reply compiler/runtime/Helpers.inc]: Yes, I think that can be deleted as well.
However, I have removed only those enums which had no presence elsewhere apart from Helpers.inc.
So, I think there might a few more similar to emilyCallGlue which have presence in other file (just in switch-case).

Comment (0xdaryl) [reply compiler/runtime/Helpers.inc]: Would you happen to remember which others?  I can quickly eyeball them and let you know if they can be deleted.

Comment (PeXXeR) [reply compiler/runtime/Helpers.inc]: TR_AbstractMethodError
TR_IllegalAccessError
PyHelper_TupleGetItem
PyHelper_BinaryRemainder
RubyHelper_vm_send_simple

And similarly other Python and Ruby helper enums.

Comment (0xdaryl) [reply compiler/runtime/Helpers.inc]: Leave the Py and Ruby helpers for now.  `TR_AbstractMethodError` and `TR_IllegalAccessError` can go.

Comment (PeXXeR) [reply compiler/runtime/Helpers.inc]: Addressed in 8d71b30

**Comment by 0xdaryl** [comment]
Can you squash these commits please and I will launch the sanity tests?

**Comment by 0xdaryl** [comment]
@genie-omr build all

**Pull request merged**